tests/testComposite.py

```diff
@@ -50,41 +65,47 @@ def tearDown(self):
         if os.path.exists(self.output):
             shutil.rmtree(self.output)
 
-    def test(self):
+    def testGet(self):
         """Verify that a composite can be loaded and that its components are the same as when the type1
         components are loaded individually (verifies correct lookup in this case).
+
         Also verify that when the individual components are put and when the composite is put (which
         disassembles into individual components) that the objects that are written are the same.
         """
+        butler = dafPersist.Butler(
+            inputs=dafPersist.RepositoryArgs(root=self.input, mapper='lsst.obs.test.testMapper.TestMapper'),
+        )
+        rawAndFlat = butler.get('rawAndFlat', dataId={'visit': 1, 'filter': 'g'})
```

Comment:
Since you use this dataId in many places here, it might be good to set it in `setUp()` as `self.rawDataId`, then you can use it here and in `testPut`